figures/S14/data/autophagy_results.csv

```diff
@@ -1,20 +1,21 @@
 Search query: autophagy
 Year,Count
-2024,385
-2023,9945
-2022,10526
-2021,10618
-2020,9651
+2025,110
+2024,10042
+2023,10094
+2022,10547
+2021,10626
+2020,9653
 2019,8272
 2018,7274
 2017,6529
-2016,5511
-2015,5017
+2016,5510
+2015,5016
 2014,4338
-2013,3688
+2013,3687
 2012,3023
 2011,2316
-2010,1874
+2010,1872
 2009,1442
```

Comment:
Similar to my comment above, but in this case, each previous year lost one publication.

Reply:
insteresting...no explaination here